uuv_world_plugins/uuv_world_ros_plugins/src/SphericalCoordinatesROSInterfacePlugin.cc

```diff
@@ -135,8 +135,13 @@ bool SphericalCoordinatesROSInterfacePlugin::TransformFromSphericalCoord(
   ignition::math::Vector3d cartVec =
     this->world->GetSphericalCoordinates()->LocalFromSpherical(scVec);
 #endif
-  _res.output.x = cartVec.X();
-  _res.output.y = cartVec.Y();
+  // The minus signs on X and Y come from the fact that
+  // Gazebo outputs the cartesian coordinates in the
+  // -E/-U/N format instead of E/N/U format
```

Comment:
Should this not be -E/N/-U (rotating about N by -180)

Reply:
Ah indeed I made a typo, it should be -E/-N/U